### PR TITLE
Created details popup

### PR DESCRIPTION
**Pull request by AyoMoses1**
I created a modal window popup to assist users to be able to see the project details in the projects section of the homepage. Please feel free to review and drop your feedbacks where necessary

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -422,7 +523,38 @@ textarea {
...
+}
+
+.live_link,
+.source_link {
+  text-decoration: none !important;
+}
+
 @media (min-width: 768px) {
+  div.modal {
+    width: 80%;
```
Comment (omarsalem7) [inline styles.css]: - [ ] Kindly decrease the modal's width to `65%` instead of `80%` to make your design match the Figma design.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @AyoMoses1 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -422,7 +523,38 @@ textarea {
...
+  div.modal_flex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  div.modal_flex div:first-child {
+    width: 60%;
+  }
+
   header {
```
Comment (tresorsawasawa) [inline styles.css:558]: - [ **OPTIONAL** ] **RECOMMENDED:** As this detail was not mentioned by the previous reviewer, I really suggest you implement it before merging your PR. Kindly make sure you fix the overalaping `See Live` and `See Source` buttons styling in mobile version.
Here is how your design looks:
Screenshot from 2023-01-04 23-41-31

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @AyoMoses1 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**